### PR TITLE
feat(RingTheory.MvPowerSeries.Topology): a power series is the sum of its homogeneous components

### PR DESCRIPTION
**Pull request by AntoineChambert-Loir**
Prove that a multivariate power series is the sum (in the sense of summable) families
of its homogeneous components.

Co-authored-by: Maria Inés de Frutos Fernandez @mariainesdff 

---

- [x] depends on: #14983 [to define the homogeneous components]
- [x] depends on: #14866 [to define the topology on mv power series]

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/PowerSeries/Order.lean [hunk 1]
```diff
@@ -137,7 +137,7 @@ theorem min_order_le_order_add (φ ψ : R⟦X⟧) : min (order φ) (order ψ) 
   refine le_order _ _ ?_
   simp +contextual [coeff_of_lt_order]
 
-@[deprecated (since := "2024-11-12")] alias le_order_add := min_order_le_order_add
+@[deprecated (since := "2024-11-30")] alias le_order_add := min_order_le_order_add
```
Comment (jcommelin) [inline Mathlib/RingTheory/PowerSeries/Order.lean:140]: This doesn't seem to do what the PR title promises. Did you intend for a larger diff?

Comment (AntoineChambert-Loir) [reply Mathlib/RingTheory/PowerSeries/Order.lean]: I changed the date when I mistakenly thought I had modified that alias myself.

**Comment by jcommelin** [comment]
@AntoineChambert-Loir This PR has an empty diff. Is there a missing file or something else?

**Comment by AntoineChambert-Loir** [comment]
It seems that during the various `merge` needed by the PR processess, I deleted the lemma that we had proved…

**Comment by AntoineChambert-Loir** [comment]
It seems that the relevant lemmas have already been merged to mathlib. Probably I made mistakes when I had to adjust the PR, and these two lemmas were merged into #14866.

**Pull request closed without merging**